Makefile: remove 'install-cython' dependency from the 'cythonize' rule

The 'install-cython' dependency invokes pip, which is not installed.
This emits an error during build.

Cython is already installed by buildroot, so there is no point to invoke
pip anyhow.

Signed-off-by: Frederik Peter Aalund <fpa@sbtinstruments.com>

--- a/Makefile    2021-02-07 15:50:53.188657279 +0100
+++ b/Makefile    2021-02-07 15:51:00.660701937 +0100
@@ -59,7 +59,7 @@ aiohttp/%.c: aiohttp/%.pyx $(call to-has
 
 
 .PHONY: cythonize
-cythonize: .install-cython $(PYXS:.pyx=.c)
+cythonize: $(PYXS:.pyx=.c)
 
 .install-deps: .install-cython $(PYXS:.pyx=.c) $(call to-hash,$(CYS) $(REQS))
        pip install -r requirements/dev.txt
